TACC/Core-Portal
# FP-1006: Compress/Extract Toolbar App Adjustments

## Overview: ##
Add padding between input field and help text on compress modal.
 
## Related Jira tickets: ##

* [FP-1006](https://jira.tacc.utexas.edu/browse/FP-1006)

## Summary of Changes: ##
- Added a scss style class for input field with bottom padding.

## Testing Steps: ##
1. Open any My Data folder in Data Files.
2. Click on any file(s) and hit the Compress button.
3. Confirm there is now space between the input field and help text under the input field on the modal.

## UI Photos:
![image](https://user-images.githubusercontent.com/43251554/134569264-bc46fce2-7f56-4535-9786-1f933b5280b1.png)

## Notes: ##
Other bug on ticket (compress/extract silent fail without separate push keys event) no longer exists since systems are now on tacc.cloud.
